### ISSUE TITLE
[OutOfSSA] ValueError: The given set of variables is not an independent set. At least two variables interfere!

### ISSUE THREAD

**Issue by mm4rks** [fkie-cad/dewolf#175]
### What happened?

```
Traceback (most recent call last):
  File "/home/user/dewolf/decompile.py", line 82, in <module>
    main(Decompiler)
  File "/home/user/dewolf/decompiler/util/commandline.py", line 87, in main
    task = decompiler.decompile(function_name, options)
  File "/home/user/dewolf/decompile.py", line 56, in decompile
    pipeline.run(task)
  File "/home/user/dewolf/decompiler/pipeline/pipeline.py", line 109, in run
    raise e
  File "/home/user/dewolf/decompiler/pipeline/pipeline.py", line 102, in run
    instance.run(task)
  File "/home/user/dewolf/decompiler/pipeline/ssa/outofssatranslation.py", line 83, in run
    self._out_of_ssa()
  File "/home/user/dewolf/decompiler/pipeline/ssa/outofssatranslation.py", line 100, in _out_of_ssa
    self.out_of_ssa_strategy[self._optimization](self)
  File "/home/user/dewolf/decompiler/pipeline/ssa/outofssatranslation.py", line 141, in _lift_minimal_out_of_ssa
    MinimalVariableRenamer(self.task, self.interference_graph).rename()
  File "/home/user/dewolf/decompiler/pipeline/ssa/variable_renaming.py", line 262, in __init__
    super().__init__(task, interference_graph)
  File "/home/user/dewolf/decompiler/pipeline/ssa/variable_renaming.py", line 95, in __init__
    self._contract_variables_that_need_same_name()
  File "/home/user/dewolf/decompiler/pipeline/ssa/variable_renaming.py", line 179, in _contract_variables_that_need_same_name
    self.interference_graph.contract_independent_set(connected_component)
  File "/home/user/dewolf/decompiler/structures/interferencegraph.py", line 62, in contract_independent_set
    raise ValueError(f"The given set of variables is not an independent set. At least two variables interfere!")
ValueError: The given set of variables is not an independent set. At least two variables interfere!

```

### How to reproduce?

```python decompile.py 7923f949e7422ac02c2dc5148950861f8102c83859f00b7d09857b95f16f7caf sub_408940 --pipeline.debug```

### Affected Binary Ninja Version(s)

3.3.3996 (Build ID e34a955e)

**Comment by NeoQuix** [comment]
Sorry my bad, #151 is NOT merged yet. 
But will be fixed when it is merged. 

Duplicate of #136

**Comment by NeoQuix** [comment]
Fixed with #150
